dist/tools/pyterm/pytermcontroller/pytermcontroller.py

```diff
@@ -33,11 +33,9 @@ def connectionMade(self):
         print("new connection made")
 
     def connectionLost(self, reason):
-        self.factory.numProtocols = self.factory.numProtocols - 1
```

Comment:
Removing this duplicate cannot go in a commit saying only "fix pep8".

It looks like the first one should indeed be remove as 'numProtocols' is not used anywhere else.
Also the same line can be seen here: http://twistedmatrix.com/documents/13.0.0/core/howto/servers.html#auto1

Reply:
ok, so the function should be renamed to `connectionMade` ?

Reply:
ok, forget about my previous comment. It's one or the other. Maybe I can merge the 2 functions.

Reply:
Its the second one that matters, I just want you to remove the duplicate in a separate commit.

Reply:
ok, done @cladmi

Reply:
Just add a mention in the commit body that it was a copy paste of the given link and that numProtocols is unused.
It took me some time to verify if it was indeed the right one to remove so better include in the commit documentation.

Reply:
done @cladmi, I hope the comment is ok now ;)